src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -197,15 +204,15 @@ bool ScanAssembly(Assembly assembly, Dictionary<string, bool> processed)
 
             if (assembly.GetName().Name == CoreAssemblyName)
```

Comment:
Feels strangs as it is not clear why this is done.

Reply:
the core assembly will be explicitly scanned by the scanning component and doesn't need to be scanned additional times.